allennlp/tests/nn/util_test.py

```diff
@@ -604,6 +606,94 @@ def test_viterbi_decode(self):
                                          observations)
         assert indices == [2, 3, 3, 0, 4, 3]
 
+    def test_viterbi_decode_top_k(self):
+        # Test cases taken from: https://gist.github.com/PetrochukM/afaa3613a99a8e7213d2efdd02ae4762
+
+        # Test Viterbi decoding is equal to greedy decoding with no pairwise potentials.
```

Comment:
Nit: Each of these comments seems to indicate a particular property to test. It would be somewhat cleaner to break them out into separate test cases.

Reply:
breaking out into multi would break the similarity with the previous test and others in the file.

Reply:
Fair point about the previous test.